admin/admin_settings.php

```diff
@@ -119,11 +123,14 @@
 
 $q = db_select("select * from brugere where brugernavn = '$brugernavn'",__FILE__ . " linje " . __LINE__);
 $r = db_fetch_array($q);
-if ($brugerId=$r['id']) {
+/** The following test is always false as this is the only occurence of $brugerId in the entire codebase.
```

Comment:
The variable bruger_id exsists, however the variables that get created don't seem to be called anywhere

Reply:
Actually, I have misread the if statement as a comparison. But now I see it is an assignment to `$brugerId`.

In this case, a more readable code would be just `isset($r['id']` as it is doing nothing more. And `$brugerId` is not used anywhere else